code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -216,3 +216,41 @@
 	new_heretic.knowledge_points = min(new_heretic.knowledge_points, 5)
 
 	return TRUE
+
+//////////////////////////////////////////////
+//                                          //
+//           LATEJOIN CHANGELING            //
+//                                          //
+//////////////////////////////////////////////
+
```

Comment:
```suggestion
/// Ruleset for latejoin changelings
```

These ascii art style comments are banned because they break autodoc and are ugly

Reply:
All of the dynamic rulesets have these ascii art styles. Do you want me to go through and fix them all after this one?

Reply:
![image](https://user-images.githubusercontent.com/65363339/231333680-aaae30ac-faf4-4013-a122-680b1c85f6e7.png)